docs/developer/order-expiration.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Order Expiration
+---
+
+:::info
+
+expireOrdersAfter was introduced in **Saleor 3.13**
+
+deleteExpiredOrdersAfter was introduced in **Saleor 3.14**
+
+:::
+
+:::caution
+
+This feature is in the **Feature Preview** stage, which means that it is available for experimentation and
+feedback. However, it is still undergoing development and is subject to modifications.
+
+:::
+
+## Introduction
+
+Saleor allows to finalize transactions without payment (order first, then payment). If the customer cancel payment process and order remains unpaid for some time, there is possibility to set automatic stock releases and remove order.
```

Comment:
IMHo `finalize transactions` doesn't say to me, that we are talking about checkout->order process. 

> If the customer cancel payment process and order remains unpaid for some time, there is possibility to set automatic stock releases and remove order.

This only one use case. There is also `allowUnpaidOrders` that would allow you to convert checkout into order without any payment by calling `checkoutComplete`. So you will have unpaid orders, that will wait for a payment or expiration